content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -525,11 +525,13 @@ This means that support is still available but will be removed in a future Kuber
 Older versions of Kubernetes also supported the following in-tree PersistentVolume types:
 
 * [`awsElasticBlockStore`](/docs/concepts/storage/volumes/#awselasticblockstore) - AWS Elastic Block Store (EBS)
-  (**not available** in v1.27)
+  (**not available** starting v1.27)
```

Comment:
👋 @adityasamant25 .
Do you think it makes sense to use a sentence to clarify the versions?

Not available in Kubernetes after v1.26.

Not available in Kubernetes v1.27 or later.

Reply:
@kbhawkey I do not think this is necessary. The text we have currently is clear.